<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogReport</class>
 <widget class="QDialog" name="DialogReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="4">
    <widget class="TextEdit" name="textEdit"/>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTextEdit" name="txtReport"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>сохранить...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnReport">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>отчет</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QComboBox" name="cmbTemplate">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>textedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cmbTemplate</sender>
   <signal>activated(int)</signal>
   <receiver>btnReport</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
